server.js

```diff
@@ -22,12 +18,75 @@ const app = express()
 app.use(cors())
 app.use(bodyParser.json())
 
-// Start defining your routes here
+// Start defining routes here
 app.get('/', (req, res) => {
   res.send('Hello world')
 })
 
+//get all films with the path "films"
+app.get('/films', (req, res) => {
+  res.json(netflixData)
+})
+
+//get film from specific year with for example the path "year/2019"
+app.get('/year/:year', (req, res) => {
+ const year = req.params.year
+ const filmsFromYear = netflixData.filter((item) => item.release_year === +year)
+ res.json(filmsFromYear)
+})
+
+//get specific film with for example the path "film/Chocolate"
+app.get('/film/:title', (req, res) => {
+  const title = req.params.title
+  const film = netflixData.filter((item) => item.title === title)
+  res.json(film)
+ })
+
+ //get the type of show with for example the path "type/Movie"
+app.get('/type/:type', (req, res) => {
+  const type = req.params.type
+  const typeOfFilm = netflixData.find((item) => item.type === type)
+  res.json(typeOfFilm)
+ })
+
+ //get the first 10 films with the path "films/firstTenFilms"
+ app.get('/films/:firstTenFilms', (req, res) => {
+  const firstTenFilms = netflixData.slice(0, 10)
+  res.json(firstTenFilms)
+ })
+
+ //http://localhost:8080/show?search=choc  this is the searchstring for shows with the word 'choc' somewhere
+ app.get('/show', (req, res) => { 
+ const titleSearchString = req.query.search;
+   let filteredShow = netflixData.filter(item => {
+     const itemTitle = item.title.toString();
+     return itemTitle.toLowerCase().includes(titleSearchString.toLowerCase())
+    
+   });
+  
+   res.json(filteredShow);
+ });
```

Comment:
This is filtering the same data as you show in /films endpoint. So make this query be available in that endpoint instead of creating a new one. So you could either add i.e `?search=choc` to the /films endpoint or not. and it will return a filtered list or the whole list based on that.